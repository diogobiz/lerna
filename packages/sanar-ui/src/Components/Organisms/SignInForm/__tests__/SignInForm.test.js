
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESSignInForm from '../'

it('renders correctly', () => {
    const component = (
        <ESSignInForm>
            <h1>ESSignInForm</h1>
        </ESSignInForm>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
