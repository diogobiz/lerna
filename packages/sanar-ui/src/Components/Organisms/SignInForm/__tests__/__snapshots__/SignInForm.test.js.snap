// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-sign-in-form"
>
  <div
    className="ant-spin-nested-loading"
  >
    <div
      className="ant-spin-container"
    >
      <form
        className="ant-form ant-form-horizontal es-form"
        onSubmit={[Function]}
      >
        <div
          className="ant-row ant-form-item es-form-item"
          rules={
            Array [
              Object {
                "message": "Este campo é obrigatório",
                "required": true,
              },
              Object {
                "message": "Este não é um e-mail válido.",
                "type": "email",
              },
            ]
          }
          style={Object {}}
        >
          <div
            className="ant-col ant-form-item-control-wrapper"
            style={Object {}}
          >
            <div
              className="ant-form-item-control"
            >
              <span
                className="ant-form-item-children"
              >
                <input
                  className="ant-input ant-input-lg es-input"
                  data-testid="es-signin-form__form__email"
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  placeholder="E-mail"
                  type="text"
                  value=""
                />
              </span>
            </div>
          </div>
        </div>
        <div
          className="ant-row ant-form-item es-form-item"
          rules={
            Array [
              Object {
                "message": "Este campo é obrigatório",
                "required": true,
                "whitespace": true,
              },
            ]
          }
          style={Object {}}
        >
          <div
            className="ant-col ant-form-item-control-wrapper"
            style={Object {}}
          >
            <div
              className="ant-form-item-control"
            >
              <span
                className="ant-form-item-children"
              >
                <span
                  className="ant-input-password es-input ant-input-password-large ant-input-affix-wrapper ant-input-affix-wrapper-lg"
                >
                  <input
                    action="click"
                    className="ant-input ant-input-lg"
                    data-testid="es-signin-form__form__password"
                    onChange={[Function]}
                    onKeyDown={[Function]}
                    placeholder="Senha"
                    style={null}
                    type="password"
                    value=""
                  />
                  <span
                    className="ant-input-suffix"
                  >
                    <i
                      aria-label="icon: eye-invisible"
                      className="anticon anticon-eye-invisible ant-input-password-icon"
                      onClick={[Function]}
                      onMouseDown={[Function]}
                      tabIndex={-1}
                    >
                      <svg
                        aria-hidden="true"
                        className=""
                        data-icon="eye-invisible"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 0 0 0-51.5zm-63.57-320.64L836 122.88a8 8 0 0 0-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 0 0 0 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 0 0 0 11.31L155.17 889a8 8 0 0 0 11.31 0l712.15-712.12a8 8 0 0 0 0-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 0 0-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 0 1 146.2-106.69L401.31 546.2A112 112 0 0 1 396 512z"
                        />
                        <path
                          d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 0 0 227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 0 1-112 112z"
                        />
                      </svg>
                    </i>
                  </span>
                </span>
              </span>
            </div>
          </div>
        </div>
        <div
          className="ant-row-flex ant-row-flex-space-between ant-row-flex-middle es-row mb-lg es-sign-in-form__form--extra"
          style={Object {}}
        >
          <div
            className="ant-col es-col"
            style={Object {}}
          >
            <label
              className="es-checkbox ant-checkbox-wrapper"
            >
              <span
                className="ant-checkbox"
                style={Object {}}
              >
                <input
                  checked={false}
                  className="ant-checkbox-input"
                  data-testid="es-signin-form__form__keep-me-logged-in"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  type="checkbox"
                />
                <span
                  className="ant-checkbox-inner"
                />
              </span>
            </label>
          </div>
          <div
            className="ant-col es-col"
            style={Object {}}
          >
            <a
              className="es-button es-button__medium es-button__variant--text primary bold ant-btn"
              data-testid="es-signin-form__form__forgot-password"
              href="/#/auth/recuperar-senha"
            />
          </div>
        </div>
        <button
          className="es-button es-button__medium es-button__variant--solid primary es-button__block bold uppercase ant-btn"
          data-testid="es-signin-form__form__do-login"
          type="submit"
        />
      </form>
    </div>
  </div>
</div>
`;
